R/apply_theme.R

```diff
@@ -9,6 +9,7 @@
 #' @param fontfamily string with the name of a supported font
 #' @param grid boolean that specifies whether the grid should be drawn or not
 #' @param bg string giving the colour for the background of the plot
+#' @param legend_position string indicating the legend position
```

Comment:
We decided to remove @author everywhere. I know we have not been consistent in this :(

Reply:
Oh, okay. Wasn't present in that meeting. Should I also remove them for the other files in this PR?

![image](https://user-images.githubusercontent.com/12818242/119939911-cd909d00-bf8e-11eb-8fc2-ae0f67392d78.png)

Reply:
Yes that is good or ask Mark to do so to avoid merge conflicts. Thanks for that